### ISSUE TITLE
Grand Theft Auto:Vice City Stories graphical issue makes game unplayable 

### ISSUE THREAD

**Issue by Winddgg** [hrydgard/ppsspp#14611]
### What happens?
Game have graphic glitch on the beginning and it's unplayable. When the game start, after the intro, the character it's falling from the sky on a never ending loop: 

![1622735867204-grand_theft_auto_vice_city_stories_eur_psp-psypsp-210603-183849](https://user-images.githubusercontent.com/87370632/125446254-de7ea212-3a3c-4361-a554-de9757ad1b0c.png)

The problem start after an update. Before it was playable. I reinstall everything to test but the results was the same. 

Verdose log from RetroPie : [https://pastebin.com/pb71K4rg](url)

### What *should* happen?
Game start normally without this error.

#### What hardware / device and operating system?
Raspberry Pi4  4gb with RetroPie 4.7.15 
#### What graphics card (GPU) or mobile phone model?
The default GPU
#### What PPSSPP version (standalone/official), and did it work before?
ir-ppsspp and the standalone ppsspp. The game was playable before, the problem start after an update.
#### Which game or games?

ULUS-10160

#### Checklist

 - [x] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
 - [x] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
 - [ ] Try resetting settings or older versions and include if the issue is related.
 - [x] Try changing graphics settings to determine if one causes the bug.
 - [x] Include logs or screenshots of issue.

**Comment by Panderner** [comment]
I can't reproduce this issue for me so it's working fine i tested on my PC and Android Phone.
![Desktop Screenshot 2021 07 14 - 10 29 31 26](https://user-images.githubusercontent.com/37503397/125551373-7ef1742e-d500-453f-bca5-de9fb4807d8b.png)

**Comment by Winddgg** [comment]
Yes is working on the PC. Maybe the issue is specific with the Pi4.

**Comment by Winddgg** [comment]
I downgrade standalone ppsspp, to version 1.10.2, and the game is running without problem.

**Comment by Panderner** [comment]
It's working fine for me in latest build.

**Comment by domb84** [comment]
I have this same issue with the rpi4 on the latest lakka nightly. it actually occurs on both Liberty City stories as well.

```https://nightly.builds.lakka.tv/latest/RPi4.arm/Lakka-RPi4.arm-nightly-20210828-497be54.tar```

**Comment by unknownbrackets** [comment]
Can you try to find the first git build with this problem?  It also helps to know the last git build that didn't have this problem.

Don't worry, you don't have to test every version to figure this out.  The trick is to go by halves.  So if you want to find where it broke among 1000 git builds, start with build 500.  If it doesn't work, go down by half - otherwise, go up by half.  With 10 tries, you can find it among 1000 builds.

For people without the game, or not experiencing the problem... they're stuck staring at each build one by one, trying to guess the problem.  That's like finding a needle in a haystack sometimes.  You're the hero that can cut down all the small fry in a few swings of a sword.

-[Unknown]
